packages/rest/src/session/doc/ISession.ts

```diff
@@ -114,6 +114,13 @@ export interface ISession {
      */
     rejectUnauthorized?: boolean;
 
+    /**
+     * The amount of time to specify in the X-IBM-Response-Timeout header
```

Comment:
is this header and implementation more specific to zowe cli & zosmf than imperative?

Reply:
The idea to add timeout to the ISession is to make it available for REST clients to be able to extend the base REST client and define a timeout as they see fit. The information provided on that comment in particular was specific to zOSMF due to the testing I was doing, and I forgot to change it to something more generic.

Reply:
Ok gotcha!